### ISSUE TITLE
Show users’ studio

### ISSUE THREAD

**Issue by mborsare** [thoughtbot/constable#460]
When a person I don't know posts a great announcement, I want to know which office they work out of so I can learn a little more about them.

**Comment by stratigos** [comment]
Would a link to their hub page satisfy this?

I agree that its useful to know which studio theyre working from, but then I wonder what other information folks would want about the announcer. That led me to thinking a link to their hub page would reveal just about anything of interest, though I then see the issue of needing an additional step to get at info (click link, view new page).

**Comment by tysongach** [comment]
I made quick mockup using a badge to show the studio people work in:

<img width="666" alt="Screen Shot 2019-09-28 at 09 53 17" src="https://user-images.githubusercontent.com/903327/65817632-c0c70f00-e1d7-11e9-974e-ab7606fc3f0c.png">

My thinking is that the badges would only exist on announcement show pages (not index pages) at the top for the announcement’s author, and also in the comments section (as shown above). Maybe the colors are too much, or not needed.

---

> I agree that its useful to know which studio theyre working from, but then I wonder what other information folks would want about the announcer.

Yeah, perhaps job title would be valuable info, as well?

---

In https://github.com/thoughtbot/constable/commit/317bf34db524ec56358d171afbc5f8972729f129 we made it possible to view announcements from a specific user. For example, here is an index of Stephanie’s announcements:

<img width="1071" alt="Screen Shot 2019-09-28 at 10 00 47" src="https://user-images.githubusercontent.com/903327/65817570-e1429980-e1d6-11e9-9355-84697b945ca4.png">

A page title/heading could be really useful on this page (“Stephanie Kuroda’s announcements”). That might pave the way to have a little header section, with the person’s name, job title and studio location, and perhaps a link to their Hub profile.
